packages/playground/src/components/SpecialInput.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+
+const COLORS = ['red', 'green', 'blue'];
+
+
+// @ts-ignore
```

Comment:
Can you fix the types here is possible and remove the `@ts-ignore`

Reply:
I used `FC<FieldProps<string>` is that the correct type for the `SpecialInput` component?